test/Interop/Cxx/function/emit-called-function-irgen.swift

```diff
@@ -0,0 +1,20 @@
+// Test that Swift emits code for a C++ inline function that is called from
```

Comment:
"Test that we emit LLVM IR for C++ inline free functions that are called directly, and also for inline functions that are called transitively. Test that we don't emit LLVM IR for inline functions that are not called."

Reply:
Thanks, this is much clearer. Done, with some additional wordsmithing that I hope you're OK with.